### PR TITLE
Screenshots have been added for Aperture Agent

### LINKED ISSUE #4842: [Screenshots] Aperture Agent under 'How it Works See It in Action' section
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/aperture-agent) has only two old screenshots of the designer and visualizer in the 'How it Works See It in Action' section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [Aperture Agent](https://layer5.io/cloud-native-management/meshery/integrations/aperture-agent) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---

<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by srujan0404**
**Description**
The screenshots have been added for Aperture Agent

This PR fixes #4842

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c84dc903e5d24935e4f0afc1699e4c922860c167 at: https://65ce79a89607ec38cbcf8759--layer5.netlify.app

**Comment by saurabh100ni** [comment]
@srujan0404 Please mention the integration name in the PR title, and it would be nice to have a cleaner screenshot (this one looks a little messy).

**Comment by srujan0404** [comment]
> @srujan0404 Please mention the integration name in the PR title, and it would be nice to have a cleaner screenshot (this one looks a little messy).

done, will not repeat from next time

**Comment by saurabh100ni** [comment]
Thanks

File: src/collections/integrations/aperture-agent/index.mdx [hunk 1]
```diff
@@ -21,8 +21,10 @@ featureList: [
   "Ongoing synchronization of Kubernetes configuration and changes across any number of clusters."
 ]
 workingSlides: [
-  ../_images/meshmap-visualizer.png,
-  ../_images/meshmap-designer.png
+  ../_images/apertureAgent/Aperture-Agent-Design-light.png,
+  ../_images/apertureAgent/Aperture-Agent-Design-dark.png,
+  ../_images/apertureAgent/Aperture-Agent-Visualize-light.png,
+  ../_images/apertureAgent/Aperture-Agent-Visualize-dark.png
```
Comment (vishalvivekm) [inline src/collections/integrations/aperture-agent/index.mdx:27]:
```suggestion

```

can be reverted.

File: src/collections/integrations/_images/apertureAgent/Aperture-Agent-Visualize-light.png
Comment (vishalvivekm) [inline src/collections/integrations/_images/apertureAgent/Aperture-Agent-Visualize-light.png:1]: please rename the dir from `apertureAgent` to `aperture-agent`.
Refer to : https://github.com/layer5io/layer5/pull/5129#issue-1995493031

**Comment by l5io** [comment]
🚀 Preview for commit 3294d8458b926b32496caecf7b50feea338db2c8 at: https://65d792e1802729262e8e3702--layer5.netlify.app

**Comment by iArchitSharma** [comment]
@srujan0404 thanks!

**Pull request merged**